v2/cf-nodejs-logging-support/src/lib/logger/root-logger.ts

```diff
@@ -75,9 +71,19 @@ export default class RootLogger extends Logger {
         options.logMessage = this.logMessage;
         return createTransport(options);
     };
+
     forceLogger(logger: framework) {
         Config.getInstance().setFramework(logger);
         RequestAccessor.getInstance().setFrameworkService();
         ResponseAccessor.getInstance().setFrameworkService();
     }
+
+    // legacy methods
+    overrideNetworkField() {
+        //to do: create a config file and override in config
```

Comment:
A config file is not require. Simply use the addConfig method to add a static field entry.